packages/communicator/src/index.ts

```diff
@@ -0,0 +1,279 @@
+/* eslint-disable */
+/* tslint:disable */
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
```

Comment:
No sure if the license heade will be found if it is not at the beginning. Can we add the eslint comments below?

Reply:
I will try to move it, this is the thing with automatic updates. I am not sure that should we do it as a workflow or just fail and force to do it like a lint task(do it locally and update it by your own...) lets discuss it.